target/linux/ath79/image/generic.mk

```diff
@@ -440,7 +440,7 @@ TARGET_DEVICES += iodata_wn-ac1600dgr
 
 define Device/iodata_wn-ac1600dgr2
   ATH_SOC := qca9557
-  DEVICE_TITLE := I-O DATA WN-AC1600DGR2
+  DEVICE_TITLE := I-O DATA WN-AC1600DGR2/DGR3
```

Comment:
I'm wondering how the potential user is going to find out, that image for DGR2 also supports DGR3 device as the firmware image naming is unchanged, thus stays named as `iodata_wn-ac1600dgr2-squashfs-factory.bin`.

Reply:
Indeed. I'll try to find better name.

Reply:
Please update to use DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT as introduced in:
https://github.com/openwrt/openwrt/commit/8c3dbaf8b49233e275b2dce4e30dc0a2bf8c86c5

Reply:
OK, I'll do it.